deployment/docker_compose/docker-compose.search-testing.yml

```diff
@@ -0,0 +1,406 @@
+version: '3'
+services:
+  api_server:
+    image: danswer/danswer-backend:latest
+    build:
+      context: ../../backend
+      dockerfile: Dockerfile
+    command: >
+      /bin/sh -c "alembic upgrade head &&
+      echo \"Starting Danswer Api Server\" &&
+      uvicorn danswer.main:app --host 0.0.0.0 --port 8080"
+    depends_on:
+      - relational_db
+      - index
+      # - inference_model_server
+    restart: always
+    ports:
+      - "8080"
+    environment:
+      # Auth Settings
```

Comment:
this can rely on an .env file explicitly. The goal is to not have to duplicate the env vars so that when we introduce a new one it has to be changed in 20 places